AllASPFuncs.py

```diff
@@ -753,14 +720,31 @@ def fetchImages(num_total_images, num_detect_images, ID, sn_path, band, size, fi
                                  roman_path=roman_path, maxbg=num_total_images - num_detect_images,
                                  maxdet=num_detect_images, return_list=True, band=band,
                                  lc_start=lc_start, lc_end=lc_end)
-    images, cutout_wcs_list, im_wcs_list, err =\
+    cutout_image_list, image_list =\
         constructImages(exposures, ra, dec, size=size,
                         background=fit_background, roman_path=roman_path)
 
+    # THIS IS TEMPORARY. In this PR, I am refactoring constructImages to return
+    # Image objects. However, the rest of the code is not refactored yet. This
+    # returns the Image objects back into the numpy arrays that the rest of the
+    # code understands.
+
+    images = []
+    cutout_wcs_list = []
+    im_wcs_list = []
+    err = []
+    for cutout, image in zip(cutout_image_list, image_list):
+        images.append(cutout._data)
+        cutout_wcs_list.append(galsim.AstropyWCS(wcs=cutout._wcs))
+        im_wcs_list.append(galsim.AstropyWCS(wcs=image._wcs))
+        err.append(cutout._noise)
+
     Lager.debug(f'here {np.shape(images)}')
```

Comment:
"here"?

Reply:
Why is this marked as "resolved"?  My question was why does the `Lager.debug` statement say "here"  (fine, fine, that's not in this PR).  Saying "here" isn't super useful, unless you have only one message -- and if that's the case you still don't need to say "here" because they're only one place..  What you really want is for the logger framework to tell you where the error was emitted.